README.md

```diff
@@ -157,12 +159,12 @@ update your Earthdata application as follows:
    **Manage > Redirect Uris**.
 1. Switch to your terminal window in your Docker container (if necessary, run
    `make docker`)
-1. Within your Docker container, run `make cumulus-tf/output` to show the
+1. Within your Docker container, run `make output-cumulus` to show the
```

Comment:
I made this change because I didn't see any command mapping to `cumulus-tf/output` in the Makefile

Reply:
Ah yes! And I noticed another one, hang on...

Reply:
In `OPERATING.md`, line 194 (I think), `make docker` should be `make bash`